modules/Bio/EnsEMBL/Hive/DBSQL/ResourceDescriptionAdaptor.pm

```diff
@@ -54,6 +54,13 @@ sub default_insertion_method {
 }
 
 
+sub overflow_limit {
+    return {
+        'submission_cmd_args' => 255,
```

Comment:
I would add `worker_cmd_args` because of the `reg_conf`, `hive_log_dir` and `worker_log_dir` topions